Editor/Scripts/GLTFImporterInspector.cs

```diff
@@ -79,6 +79,7 @@ private void ModelInspectorGUI()
 			EditorGUILayout.LabelField("Scene", EditorStyles.boldLabel);
 			EditorGUILayout.PropertyField(serializedObject.FindProperty(nameof(GLTFImporter._removeEmptyRootObjects)));
 			EditorGUILayout.PropertyField(serializedObject.FindProperty(nameof(GLTFImporter._scaleFactor)));
+			EditorGUILayout.PropertyField(serializedObject.FindProperty(nameof(GLTFImporter._importCamera)));
```

Comment:
I think this PR might be based on the camera import PR, is that intended? We might want to rebase once the camera PR has landed to make it more clear